Random.py

```diff
@@ -46,21 +46,4 @@ def print_students(self):
 manager = StudentManager()
 manager.add_student("Alice", 20, 1)
```

Comment:
### Review Comments - The indentation is inconsistent for the `if` statements. The `if`, `elif`, and `else` should align with their respective parent block for better readability and to avoid potential indentation errors. diff -      if direction == "up": +     if direction == "up": -         y -= 1 +         y -= 1 -      elif direction == "down": +     elif direction == "down": -         y += 1 +         y += 1 -      elif direction == "left": +     elif direction == "left": -         x -= 1 +         x -= 1 -      elif direction == "right": +     elif direction == "right": -         x += 1 +         x += 1